mmv1/third_party/terraform/tests/resource_vmwareengine_cluster_test.go.erb

```diff
@@ -0,0 +1,179 @@
+<% autogen_exception -%>
+package google
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+	"strings"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccVmwareengineCluster_vmwareEngineClusterUpdate(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"region":        	 "southamerica-west1", // using region with low node utilization.
+		"org_id":          acctest.GetTestOrgFromEnv(t),
+		"billing_account": acctest.GetTestBillingAccountFromEnv(t),
+		"random_suffix":   RandString(t, 10),
+	}
+
+  config := vmwareEngineClusterConfig(context)
+
+	VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: ProtoV5ProviderBetaFactories(t),
+		ExternalProviders: map[string]resource.ExternalProvider{
+			"time":   {},
+		},
+		CheckDestroy: testAccCheckVmwareengineClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(config, 3),
+			},
+			{
+				ResourceName:            "google_vmwareengine_cluster.vmw-engine-ext-cluster",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "name"},
+			},
+      {
+        Config: fmt.Sprintf(config, 4), // expand the cluster
+      },
+      {
+				ResourceName:            "google_vmwareengine_cluster.vmw-engine-ext-cluster",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "name"},
+			},
+			{
+        Config: fmt.Sprintf(config, 3), // shrink the cluster.
+      },
+      {
+				ResourceName:            "google_vmwareengine_cluster.vmw-engine-ext-cluster",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"parent", "name"},
+			},
+		},
+	})
+}
+
+func vmwareEngineClusterConfig(context map[string]interface{}) string {
```

Comment:
Not a blocker, but generally we prefix functions like this with `test` for clarity.

```suggestion
func testVmwareEngineClusterConfig(context map[string]interface{}) string {
```

Reply:
done